src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -27,36 +31,40 @@ class ReviewPage extends React.Component {
     focusElement('h2');
   }
 
+  renderDowntime = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return <Message downtime={downtime} />;
+    }
+
+    return children;
+  };
+
   render() {
-    const { formConfig, pageList, path } = this.props;
+    const { formConfig, pageList, path } = this.props.route;
 
+    const downtimeDependencies = formConfig?.downtime?.dependencies || [];
     return (
       <div>
         <ReviewChapters formConfig={formConfig} pageList={pageList} />
-        <SubmitController
-          formConfig={formConfig}
-          pageList={pageList}
-          path={path}
-        />
+        <DowntimeNotification
+          appTitle="application"
+          render={this.renderDowntime}
+          dependencies={downtimeDependencies}
+          customText={formConfig.customText}
+        >
+          <SubmitController
+            formConfig={formConfig}
+            pageList={pageList}
+            path={path}
+          />
+        </DowntimeNotification>
```

Comment:
Wrap the `SubmitController` in a `DowntimeNotification`, just like `RoutedSavableReviewPage`:

https://github.com/department-of-veterans-affairs/vets-website/blob/d412bc0a1834ff507761f4be7f4d94eebb876349/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L90-L101